AdyenDropIn/DropInComponent.swift

```diff
@@ -265,12 +261,6 @@ public final class DropInComponent: NSObject,
         (component as? PreApplePayComponent)?.presentationDelegate = self
         
         component._isDropIn = true
-        
-        if let payment = configuration.payment, let remainingAmount = component.order?.remainingAmount {
-            component.payment = Payment(amount: remainingAmount, countryCode: payment.countryCode)
-        } else {
-            component.payment = configuration.payment
-        }
```

Comment:
where is the code that changes the Payment amount in the AdyenContext in case of partial payment order remaining amount? am I missing something?

Reply:
It is moved to [ComponentManager](https://github.com/Adyen/adyen-ios/pull/940/files/3d61fcfa7558cbfca94d3410eae0dd859d75eef0#diff-fd23d788c9a3e6dec6b94258dbabae37abd2de363812a97a822efe48678d81e3R48-R54)

Reply:
[will continue the discussion here](https://github.com/Adyen/adyen-ios/pull/940#discussion_r919929556) :-)